injection/injection.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package injection
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/signals"
+
+	"k8s.io/client-go/rest"
+)
+
+// EnableInjectionOrDie enables Knative Injection and starts the informers.
+// Both Context and Config are optional. Returns context with rest config set
+// and a function to start the informers after watches have been set up.
+func EnableInjectionOrDie(ctx context.Context, cfg *rest.Config) (context.Context, func()) {
```

Comment:
This makes sense to me, though the name is a little bit confusing as the original `EnableInjectionOrDie` is exported too.

Reply:
yeah, that is intended, the original one is in a different package and marked deprecated.